lib/active_support/cache/memcached_store.rb

```diff
@@ -13,11 +13,14 @@ module Cache
     class MemcachedStore < Store
       ESCAPE_KEY_CHARS = /[\x00-\x20%\x7F-\xFF]/n
 
-      attr_accessor :read_only
+      attr_accessor :read_only, :swallow_exceptions
 
       def initialize(*addresses)
         addresses = addresses.flatten
         options = addresses.extract_options!
+        @swallow_exceptions = true
+        @swallow_exceptions = options.delete(:swallow_exceptions) if options.key?(:swallow_exceptions)
```

Comment:
`@swallow_exceptions` would then be `nil`, wouldn't it? And since we raise `unless @swallow_exceptions` I think it would be different. Or is it too early in the morning for me still?